nixos/modules/system/boot/networkd.nix

```diff
@@ -1941,13 +1928,7 @@ in
       # networkd.
       systemd.sockets.systemd-networkd.wantedBy = [ "sockets.target" ];
 
-      systemd.services.systemd-networkd = {
-        wantedBy = [ "multi-user.target" ];
-        aliases = [ "dbus-org.freedesktop.network1.service" ];
-        restartTriggers = map (x: x.source) (attrValues unitFiles) ++ [
-          config.environment.etc."systemd/networkd.conf".source
-        ];
-      };
+      systemd.services.systemd-networkd.aliases = [ "dbus-org.freedesktop.network1.service" ];
```

Comment:
Do we also have dbus in stage1? :eyes:

Reply:
We do not. But networkd runs fine without it

Reply:
So should this alias not be in here? Or should it be? Or it doesn't matter?

Reply:
We should probably at least have a comment explaining this line. I was quite surprised to see dbus stuff in stage1.

Reply:
I was trying to leave as much code in common as the stage 1 systemd options currently support. No rhyme or reason for this in particular to be in stage 1.